defaults/main.yml

```diff
@@ -1,17 +1,17 @@
 ---
-openio_filebeat_version: 6.7.0
-
-openio_filebeat_namespace: OPENIO
-openio_filebeat_gridinit_dir: "/etc/gridinit.d/{{ openio_filebeat_namespace }}"
-openio_filebeat_gridinit_file_prefix: ""
-openio_filebeat_pid_directory:
-  "/run/filebeat/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_version: 7.5.0
 
 openio_filebeat_serviceid: "0"
-openio_filebeat_volume: "/var/lib/oio/sds/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_namespace: OPENIO
+openio_filebeat_gridinit_dir: "/etc/gridinit.d"
+openio_filebeat_gridinit_file_prefix: "{{ openio_filebeat_namespace }}-"
```

Comment:
openio_filebeat_gridinit_dir: "/etc/gridinit.d/{{ openio_filebeat_namespace }}"
openio_filebeat_gridinit_file_prefix: ""

Reply:
why the (hell) all the roles are creating the directory `/etc/gridinit.d/{{ namespace }}` and the gridinit service files are then stored in the directory `/etc/gridinit.d` and named `{{ namespace }}-{{ service_name }}-{{ service_id }}.conf` ?

this is non sens

Reply:
fixed